android/libraries/rib-coroutines/src/main/kotlin/com/uber/rib/core/RibCoroutines.kt

```diff
@@ -0,0 +1,20 @@
+package com.uber.rib.core
+
+import com.uber.autodispose.ScopeProvider
+import com.uber.autodispose.coroutinesinterop.asCoroutineScope
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.SupervisorJob
+
+val ScopeProvider.defaultScope: CoroutineScope
```

Comment:
Looking for feedback on 1 scope per default dispatcher as extension functions to interactor.

Reply:
Discussed offline, using one scope since there's one lifecycle and the caller can specify other dispatchers in launch command.